providers/resource/jobs/resource.pxu

```diff
@@ -476,3 +476,13 @@ _summary: Collect information about the audio card
 _description: Gets audio resource info from /proc/asound/pcm
 command: audio_card_resource.py
 
+id: image_source_and_type
+estimated_duration: 0.05
+plugin: resource
+_summary: Collect the source and type of image
+_description:
+    Get the source and type of image.
+    Source: 'stock' or 'oem'
+    Type: 'classic' or 'core'
+command: checkbox-support-image_checker -t -s
+flags: preserve-locale
```

Comment:
Is there a reason to use `preserve-locale` for this job?

Reply:
It was an accident hahaha. It should not be there. I've removed it.